beetmoverscript/src/beetmoverscript/task.py

```diff
@@ -34,9 +36,39 @@ def validate_task_schema(context):
     client.validate_task_schema(context, schema_key=schema_key)
 
 
+def _get_bucket_prefixes(script_config):
+    return _get_scope_prefixes(script_config, "bucket")
+
+
+def _get_action_prefixes(script_config):
+    return _get_scope_prefixes(script_config, "action")
```

Comment:
is it an easy migration? Worth doing here? We've got some time still

Reply:
The steps would probably be:

- support setting the action/behavior from a task payload key/value pair
  - add that to the schema, look there first, fall back to the scope
  - roll that out to production beetmover
- change the in-tree configs to use the task payload instead
- repeat for all branches and repos that use the scope to set the action/behavior
- once all uses of action scopes are in the task payload instead of scope, remove the scope and support for reading the action from the scope

If we introduced the action scope in this PR, that would be pretty easy. I suspect we're using beetmover action scopes all over gecko and other repos, so the in-tree portion might be lengthy.

Reply:
let's punt on moving away from action scopes. sometime in the unplanned future.